releasenotes/notes/low-rank-linalg-86da86096d6122d8.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Added linear algebra utilities for performing the double-factorization of a two-body tensor:
+    - :func:`qiskit_nature.utils.double_factorized`
+    - :func:`qiskit_nature.utils.modified_cholesky`
```

Comment:
This does not come out as a sub-list. Maybe needs a space

![image](https://user-images.githubusercontent.com/40241007/235686983-bf256445-3a46-4874-918c-3a772a112101.png)

Reply:
I added a two-space indent, hopefully that fixes it.